CI fails with dev dependencies: TypeError: GRPOTrainer._get_per_token_logps_and_entropies() got an unexpected keyword argument 'mm_token_type_ids'

CI fails with dev dependencies: https://github.com/huggingface/trl/actions/runs/22391386434/job/64814011789 

> TypeError: GRPOTrainer._get_per_token_logps_and_entropies() got an unexpected keyword argument 'mm_token_type_ids'

```python
  FAILED tests/test_grpo_trainer.py::TestGRPOTrainer::test_training_vlm_beta_non_zero[trl-internal-testing/tiny-Qwen2_5_VLForConditionalGeneration] - TypeError: GRPOTrainer._get_per_token_logps_and_entropies() got an unexpected keyword argument 'mm_token_type_ids'
  FAILED tests/test_grpo_trainer.py::TestGRPOTrainer::test_training_vlm_and_importance_sampling[trl-internal-testing/tiny-Qwen2_5_VLForConditionalGeneration] - TypeError: GRPOTrainer._get_per_token_logps_and_entropies() got an unexpected keyword argument 'mm_token_type_ids'
  FAILED tests/test_rloo_trainer.py::TestRLOOTrainer::test_training_vlm[trl-internal-testing/tiny-Qwen2_5_VLForConditionalGeneration] - TypeError: RLOOTrainer._get_per_token_logps_and_entropies() got an unexpected keyword argument 'mm_token_type_ids'
  FAILED tests/test_rloo_trainer.py::TestRLOOTrainer::test_training_vlm[trl-internal-testing/tiny-Qwen2VLForConditionalGeneration] - TypeError: RLOOTrainer._get_per_token_logps_and_entropies() got an unexpected keyword argument 'mm_token_type_ids'
  FAILED tests/test_rloo_trainer.py::TestRLOOTrainer::test_training_vlm_beta_non_zero[trl-internal-testing/tiny-Qwen2_5_VLForConditionalGeneration] - TypeError: RLOOTrainer._get_per_token_logps_and_entropies() got an unexpected keyword argument 'mm_token_type_ids'
  FAILED tests/test_rloo_trainer.py::TestRLOOTrainer::test_training_vlm_peft[trl-internal-testing/tiny-Qwen2_5_VLForConditionalGeneration] - TypeError: RLOOTrainer._get_per_token_logps_and_entropies() got an unexpected keyword argument 'mm_token_type_ids'
  FAILED tests/test_rloo_trainer.py::TestRLOOTrainer::test_training_vlm_multi_image[trl-internal-testing/tiny-Qwen2_5_VLForConditionalGeneration] - TypeError: RLOOTrainer._get_per_token_logps_and_entropies() got an unexpected keyword argument 'mm_token_type_ids'
```

Stacktrace:
```python
>       trainer.train()

../trl/tests/test_grpo_trainer.py:1806: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
src/transformers/trainer.py:1424: in train
    return inner_training_loop(
src/transformers/trainer.py:1506: in _inner_training_loop
    self._run_epoch(
src/transformers/trainer.py:1732: in _run_epoch
    tr_loss_step = self.training_step(model, inputs, num_items_in_batch)
../trl/trl/trainer/grpo_trainer.py:1090: in training_step
    output = super().training_step(model, inputs, num_items_in_batch)
src/transformers/trainer.py:1898: in training_step
    inputs = self._prepare_inputs(inputs)
../trl/trl/extras/profiling.py:202: in wrapper
    return func(self, *args, **kwargs)
../trl/trl/trainer/grpo_trainer.py:1119: in _prepare_inputs
    generation_batch = self._generate_and_score_completions(generation_batch)
../trl/trl/trainer/grpo_trainer.py:1760: in _generate_and_score_completions
    ref_per_token_logps, _ = self._get_per_token_logps_and_entropies(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <trl.trainer.grpo_trainer.GRPOTrainer object at 0x7f8b3c49d270>
args = (Qwen2_5_VLForConditionalGeneration(
  (model): Qwen2_5_VLModel(
    (visual): Qwen2_5_VisionTransformerPretrainedMode...1, 1, 1, 1, 1,
         1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]],
       device='cuda:0'), 8)
kwargs = {'batch_size': 3, 'image_grid_thw': tensor([[1, 6, 4],
        [1, 6, 4],
        [1, 6, 4]], device='cuda:0'), 'mm_to... 1, 1, 1, 0, 0, 0,
         0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]], device='cuda:0'), 'num_images': [1, 1, 1], ...}

    @functools.wraps(func)
    def wrapper(self, *args, **kwargs):
        # Check if self is a Trainer-like object with required attributes
        if hasattr(self, "state") and hasattr(self, "args"):
            with profiling_context(self, func.__name__):
>               return func(self, *args, **kwargs)
E               TypeError: GRPOTrainer._get_per_token_logps_and_entropies() got an unexpected keyword argument 'mm_token_type_ids'

../trl/trl/extras/profiling.py:202: TypeError
```

--- a/trl/trainer/grpo_trainer.py
+++ b/trl/trainer/grpo_trainer.py
@@ -1027,6 +1027,7 @@ class GRPOTrainer(BaseTrainer):
         pixel_attention_mask=None,
         image_sizes=None,
         token_type_ids=None,
+        mm_token_type_ids=None,
     ) -> dict[str, torch.Tensor | None]:
         """Compute log-probs and (optionally) entropies for each token."""
         batch_size = batch_size or input_ids.size(0)  # Chunk inputs into smaller batches to reduce memory peak
@@ -1056,6 +1057,8 @@ class GRPOTrainer(BaseTrainer):
                 model_inputs["image_sizes"] = image_sizes[start : start + batch_size]
             if token_type_ids is not None:
                 model_inputs["token_type_ids"] = token_type_ids[start : start + batch_size]
+            if mm_token_type_ids is not None:
+                model_inputs["mm_token_type_ids"] = mm_token_type_ids[start : start + batch_size]
 
             # Only add logits_to_keep if the model supports it
             if "logits_to_keep" in self.model_kwarg_keys:

--- a/trl/trainer/rloo_trainer.py
+++ b/trl/trainer/rloo_trainer.py
@@ -696,6 +696,7 @@ class RLOOTrainer(BaseTrainer):
         pixel_attention_mask=None,
         image_sizes=None,
         token_type_ids=None,
+        mm_token_type_ids=None,
     ) -> dict[str, torch.Tensor | None]:
         """Compute log-probs and (optionally) entropies for each token."""
         batch_size = batch_size or input_ids.size(0)  # Chunk inputs into smaller batches to reduce memory peak
@@ -725,6 +726,8 @@ class RLOOTrainer(BaseTrainer):
                 model_inputs["image_sizes"] = image_sizes[start : start + batch_size]
             if token_type_ids is not None:
                 model_inputs["token_type_ids"] = token_type_ids[start : start + batch_size]
+            if mm_token_type_ids is not None:
+                model_inputs["mm_token_type_ids"] = mm_token_type_ids[start : start + batch_size]
 
             # Only add logits_to_keep if the model supports it
             if "logits_to_keep" in self.model_kwarg_keys:

